docs/DeveloperGuide.md

```diff
@@ -171,7 +171,7 @@ Given below is an example usage scenario and how the create event mechanism beha
 
 Step 1:
 The user launches the application and enters the command to create a new event.
-`ecreate n/Clean up the beach r/cleaner sd/23/10/23 1900 l/East Coast Park dsc/Pick up all litter on the beach m/10 trash bags b/50.00`. After parsing all the arguments, a new `Event` object is created along with its respective fields.
+`ecreate n/Clean up the beach r/10 cleaner sd/23/10/23 1900 l/East Coast Park dsc/Pick up all litter on the beach m/10 trash bags b/50.00 vs/20`. After parsing all the arguments, a new `Event` object is created along with its respective fields.
 <puml src="diagrams/EventClassDiagram.puml" alt="EventClassDiagram" />
```

Comment:
perhaps can try using an object diagram to be used here, with all the fields specified. however, leave the event class diagram be since the my changes to the Design portion above uses it. you can base the object diagram off of how you did the class diagram

Reply:
if u wanna can check my PR `styleFields.puml` file to create object diagrams with fields specified

Reply:
that's a fantastic idea tbh

Reply:
yum yum!